packages/ui/src/PageHeader.tsx

```diff
@@ -97,3 +101,19 @@ PageHeader.Image = function PageHeaderImage({
     </div>
   )
 }
+
+function PageHeaderDescription({
+  description,
+}: {
+  description: NonNullable<PageHeaderDescriptionProp>
+}) {
+  if (typeof description === 'object' && 'text' in description) {
+    return (
+      <DescriptionText isClamped={description.isClamped}>
+        {description.text}
+      </DescriptionText>
+    )
+  }
+
+  return <DescriptionText>{description}</DescriptionText>
+}
```

Comment:
@CharlyMartin @barbaraperic - Do we still need this now that `isClamped` has been moved?

Reply:
As discussed, let's simplify to only accept the object prop.